app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/channels/ChannelTile.kt

```diff
@@ -22,6 +22,8 @@ data class ChannelTile(
     val url: String,
     val title: String,
     val setImage: (ImageView) -> Unit,
+    // tileSource should not be used to change UI, which should be generic. It
+    // is currently only used for telemetry
     val tileSource: TileSource,
```

Comment:
nit: I wonder if we should mark this deprecated so others get a red-flag when they try to add code for it.

Alternatively, we could call this, "telemetryTileSource" or something so others think twice before using it.

Reply:
This is unrelated to this issue and we can always fix it elsewhere: going to resolve this.